examples/install/cluster-operator/05-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -32,16 +32,16 @@ spec:
               value: strimzi/kafka-connect-s2i:latest
             - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
               value: strimzi/topic-operator:latest
-            - name: STRIMZI_DEFAULT_INIT_KAFKA_IMAGE
-              value: strimzi/init-kafka:latest
+            - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+              value: strimzi/kafka-init:latest
             - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
               value: strimzi/zookeeper-stunnel:latest
             - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
               value: strimzi/kafka-stunnel:latest
             - name: STRIMZI_DEFAULT_TLS_SIDECAR_TOPIC_OPERATOR_IMAGE
               value: strimzi/topic-operator-stunnel:latest
             - name: STRIMZI_LOG_LEVEL
-              value: INFO
+              value: DEBUG
```

Comment:
Out of curiosity, isn't Debug too much?

Reply:
Damn. Good that you noticed it. It is just a leftover from my private configuration which I forgot to remove before committing. Thanks.